news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,143 @@
+
+The difficulty value of colour can now be calculated using this new encoding information. This means that the colour difficulty of each note can be defined as the amount of information it contributes to the sequence so far. Generally, the more information-dense a given colour sequence is, the harder it is to decode and play. We can estimate the information density of a given pattern by the amount of encoding chunks required to describe the pattern. Finally, we try to restrict colour difficulty to consider past notes only. Hence, sequences that share the same beginning should have identical difficulty values, up until the point where they differ.
+
+### Calculating the final result
+
+Given the three skills briefly described above (stamina, rhythm and colour), their *strain* values have now become the number that approximates star rating. Switching from the previous system, where strain and individual skill values formed star rating, strain is now the only summative factor in this calculation via the new *peak* skill.
+
+ *Peak* difficulty splits the map into sections and combines the highest strains of each skill in each section, aiming to catch out particularly demanding moments. The final star rating is the outputted result of the peak skill, including a slight multiplier to scale better. 
+
+While this is much more mathematically complex than described here, you can consult the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit), or dive into the code over in the [osu!(lazer) GitHub repository](https://github.com/ppy/osu) for the full details.
+
+While previous sentiments aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/en/Ranking_Criteria/Difficulty_naming#osu!taiko), this rework has deviated away from that goal, aiming to reflect true difficulty, regardless of naming. This also means that star rating values for maps with near-FCs are no longer near 9.5 stars, but rather can expand well into the 10 star range (medal hunters rejoice!).
+
+## Changes to performance points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values (around 40% lower). In addition, the survey indicated that the way performance points are calculated needed to be re-evaluated.
+
+Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance not only between difficulty-increasing mods, but their reductive counterparts.
+
+Served alongside the star rating rework, these changes aim to accurately reflect the new sentiments of the star rating system.
+
+### Globally applied changes
```

Comment:
what happened to the [global pp multiplier](https://github.com/ppy/osu-wiki/pull/7883/files#diff-fb2c160439d756ca356ece6fdcbf35326417246ae61951409c5f5f81caf4b6eaR53)

(i also hope i don't have to double-check these numbers <.<)